tests/template/t7997.nim

```diff
@@ -0,0 +1,46 @@
+discard """
+  output: Valid and not defined
```

Comment:
You need quotes around this.

Reply:
Ooops, fixed.

Reply:
't7997' is a bad test name. Name it 'tdefined' or similiar.

Reply:
Done.